app/src/test/java/com/example/memeapp/data/mapper/MemeMapperTest.kt

```diff
@@ -36,6 +39,40 @@ class MemeMapperTest {
         assertEquals(actual, MEMES_ENTITY)
     }
 
+    @Test
+    fun `transform memes entity to realm Test`() {
+        val meme = MEMES_ENTITY.transformEntityToRealm()
+        assertEquals(MemesConstantsUtils.ID, meme.id)
+        assertEquals(MemesConstantsUtils.NAME, meme.name)
+        assertEquals(MemesConstantsUtils.BOTTOM_TEXT, meme.bottomText)
+        assertEquals(MemesConstantsUtils.IMAGE, meme.image)
+        assertEquals(MemesConstantsUtils.TAG, meme.tag)
+        assertEquals(MemesConstantsUtils.RANK, meme.rank)
+        assertEquals(MemesConstantsUtils.TOP_TEXT, meme.topText)
+    }
```

Comment:
you can use a method to reuse the functionality

Reply:
Done thx